Keas.Mvc/Services/BigfixService.cs

```diff
@@ -66,12 +66,19 @@ public async Task<BigfixComputerSearchResult[]> GetComputersByName(string name)
 
         public async Task<BigFixComputerProperties> GetComputer(string id)
         {
-            using (var bf = GetClient())
+            try {
+                using (var bf = GetClient())
+                {
+                    var results = await bf.Computers.Get(id);
+
+                    return new BigFixComputerProperties(results);
+                }
+            } 
+            catch (Exception)

```

Comment:
Do no swallow exceptions inside of a service.  Other callers may also use this service and should have the option of catching specific errors

Reply:
fixed